lib/zendesk_api/collection.rb

```diff
@@ -320,6 +320,8 @@ def to_s
       end
     end
 
+    alias to_str to_s
+
```

Comment:
Used to be 

```
alias :to_str :to_s
```

then I removed it, and now I'm re-introducing it the right way